src/marine/hgodas_insitu2ioda.py

```diff
@@ -19,13 +19,14 @@
     IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
 sys.path.append(str(IODA_CONV_PATH.resolve()))
 
-import ioda_conv_ncio as iconv
+import ioda_conv_engines as iconv
 from orddicts import DefaultOrderedDict
 
 
 vName = {
-    2210: "sea_water_temperature",
-    2220: "sea_water_salinity",
+    #          var name,            units
+    2210: ["sea_water_temperature", "???"],
+    2220: ["sea_water_salinity", "PSU"]
```

Comment:
Is "PSU" the correct units value for this variable? (I simply copied from the ioda-v1 version of this script.) Thanks!